pkg/virt-controller/watch/drain/disruptionbudget/disruptionbudget_test.go

```diff
@@ -317,6 +317,25 @@ var _ = Describe("Disruptionbudget", func() {
 			testutils.ExpectEvent(recorder, disruptionbudget.SuccessfulDeletePodDisruptionBudgetReason)
 		})
 
+		DescribeTable("should remove the pdb if the VMI reached Final state", func(vmiPhase v1.VirtualMachineInstancePhase) {
+			vmi := nonMigratableVirtualMachine()
+			vmi.Spec.EvictionStrategy = newEvictionStrategyLiveMigrate()
+			addVirtualMachine(vmi)
+			pdb := newPodDisruptionBudget(vmi, 1)
+			pdbFeeder.Add(pdb)
+
+			controller.Execute()
```

Comment:
I don't think we need to run the controller sync loop twice, why don't you just add the pdb to the `pdbSource`, make the VMI final and add it to the vmiFeeder? This should allow you to just run the controller once.

Reply:
got some weird timing issues when i used `pdbSource.Add(pdb) ` so I stayed with `pdbFeeder.Add(pdb) `

Reply:
Just checking in - is there anything else needed to get the LGTM label? Thanks!

Reply:
The `pdbSource.Add(pdb)` did not work because you just queue the object to be distributed to the `store.Cache`. The `*feeder` does actually synchronize it by waiting for the object to be distributed.

Reply:
Thanks for the explanation. I really appreciate your help!